optaplanner-examples/src/main/resources/org/optaplanner/examples/tsp/benchmark/tspBenchmarkConfig.xml

```diff
@@ -47,7 +47,7 @@
         <unionMoveSelector>
           <changeMoveSelector/>
           <swapMoveSelector>
-            <cacheType>PHASE</cacheType>
+            <cacheType>STEP</cacheType>
```

Comment:
@ge0ffrey could you please shortly summarize why TSP needs this change of cacheType?

Reply:
Yes, this is the trade-off I mentioned earlier: we can no longer do phase caching on chained changeMoves and chained swapMoves: now that fails at bootstrap instead of working. (In the past it had bugs on other types and those now fail fast during bootstrap too).
The benefit of this is that we can use those chained change moves and swap moves in multithreaded solving (just like all other moves).

I 've mentioned this in the upgrade recipe: https://github.com/kiegroup/optaplanner-website/commit/b32be369bb380f48129a4d7e9cd490678af032d5

Reply:
Thanks!